crypto/keyring/codec.go

```diff
@@ -11,11 +11,13 @@ func init() {
 }
 
 // RegisterLegacyAminoCodec registers concrete types and interfaces on the given codec.
+// TODO how to remove Info entirely?
+// rename to LegacyInfo
 func RegisterLegacyAminoCodec(cdc *codec.LegacyAmino) {
-	cdc.RegisterInterface((*Info)(nil), nil)
+	cdc.RegisterInterface((*LegacyInfo)(nil), nil)
 	cdc.RegisterConcrete(hd.BIP44Params{}, "crypto/keys/hd/BIP44Params", nil)
-	cdc.RegisterConcrete(localInfo{}, "crypto/keys/localInfo", nil)
-	cdc.RegisterConcrete(ledgerInfo{}, "crypto/keys/ledgerInfo", nil)
-	cdc.RegisterConcrete(offlineInfo{}, "crypto/keys/offlineInfo", nil)
-	cdc.RegisterConcrete(multiInfo{}, "crypto/keys/multiInfo", nil)
+	cdc.RegisterConcrete(legacyLocalInfo{}, "crypto/keys/LocalInfo", nil)
+	cdc.RegisterConcrete(legacyLedgerInfo{}, "crypto/keys/LedgerInfo", nil)
+	cdc.RegisterConcrete(legacyOfflineInfo{}, "crypto/keys/OfflineInfo", nil)
+	cdc.RegisterConcrete(legacyMultiInfo{}, "crypto/keys/MultiInfo", nil)
```

Comment:
We cannot change the 2nd argument string, it's used by amino.

Reply:
Sure